modules/apps/client-extension/client-extension-web/src/main/java/com/liferay/client/extension/web/internal/type/deployer/CETDeployerImpl.java

```diff
@@ -200,7 +230,7 @@ private ServiceRegistration<Portlet> _registerPortlet(
 
 			if (customElementCET.isUseESM()) {
 				for (int i = 0; i < urlsArray.length; i++) {
-					urlsArray[i] = "module:" + urlsArray[i];
+					urlsArray[i] = "module-shim:" + urlsArray[i];
```

Comment:
This keeps us safe from the shim errors we got in the past when all modules in DXP were using it. So, the rest of DXP module loading is not affected by this right?

Reply:
Yes. From now on we can use `module-shim` as the `<script>` type when we want the shim to be applied. In this PR we only use it for iframe, custom element, and js package client extensions, and prevent the rest of the portal from using it for anything else.